pygmt/tests/baseline/test_ternary.png.dvc

```diff
@@ -0,0 +1,4 @@
+outs:
+- md5: 9f6b57e9dd82aa24a8a013176ba8fb1d
+  size: 128744
```

Comment:
Make sure to update the plot with the new pen.

Reply:
Updated in 33d4cb0